web.tf

```diff
@@ -1,5 +1,5 @@
 
-# Our default security group to access..
+# Our default security group to access..dd
 # the instances over SSH and HTTP
```

Comment:
**Accurics remediation**
 
**[HIGH]: http port open to internet**
 
**Vulnerability**
http port open to internet
 
**Remediation**
Change cidr_block to non 0.0.0.0/0

source: accurics-onprem